bug: command not found

After cloning this repo, I ran the provided tftools testcases to get following errors

```
(py3) sampathm-macbookpro2:tests sampathm$ python tests.py 
../tftools.py external_http_lb_mig_backend_custom_header.tf  < user_inputs
../tftools.py: line 16: 
Simple wrapper tool for convert2tf and convert2rb!
: command not found
../tftools.py: line 17: import: command not found
from: can't read /var/mail/termcolor
../tftools.py: line 20: import: command not found
../tftools.py: line 21: import: command not found
../tftools.py: line 24: syntax error near unexpected token `('
../tftools.py: line 24: `def main():'
FE..
======================================================================
ERROR: test_convert2tf (__main__.TestTFtools)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 73, in test_convert2tf
    os.rename(os.path.abspath(file) + "_check", os.path.abspath(file))
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sampathm/devbox/github/terraform-sample-tools/tools/magic-modules-tftools/tests/external_http_lb_mig_backend_custom_header.tf.erb_check' -> '/Users/sampathm/devbox/github/terraform-sample-tools/tools/magic-modules-tftools/tests/external_http_lb_mig_backend_custom_header.tf.erb'

======================================================================
FAIL: test_convert2erb (__main__.TestTFtools)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 56, in test_convert2erb
    self.assertEqual(
AssertionError: 512 != 0 : Error: Failed to generate .tf.erb & .yaml files

----------------------------------------------------------------------
Ran 4 tests in 0.058s

FAILED (failures=1, errors=1)

```

--- a/tools/magic-modules-tftools/tftools.py
+++ b/tools/magic-modules-tftools/tftools.py
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3.8
 # Copyright 2021 Google LLC
 #
 # Licensed under the Apache License, Version 2.0 (the "License");

